Formula/hbase.rb

```diff
@@ -13,38 +13,35 @@ class Hbase < Formula
   end
 
   depends_on "ant" => :build
-  # 64 bit is required because of three things:
-  # the lzo jar has a native extension
-  # building native extensions requires a version of java that matches the architecture
-  # there is no 32 bit version of java for macOS since Java 1.7, and 1.8 is required for hbase
-  depends_on :arch => :x86_64
-  depends_on :java => "1.8"
   depends_on "lzo"
+  depends_on "openjdk@11"
 
   resource "hadoop-lzo" do
     url "https://github.com/cloudera/hadoop-lzo/archive/0.4.14.tar.gz"
     sha256 "aa8ddbb8b3f9e1c4b8cc3523486acdb7841cd97c002a9f2959c5b320c7bb0e6c"
+
+    patch do
+      url "https://gist.githubusercontent.com/jonchang/8fea9fca3542a89c760ca5895ea240cc/raw/82728ee23f34b46d7f14fd598d3181437ebc73e8/patch.diff"
```

Comment:
Can we use formula-patches if this works?

Reply:
Yea, just trying to figure out CI